lbrynet/reflector/client/client.py

```diff
@@ -250,7 +250,7 @@ def send_descriptor_info(self):
     def skip_missing_blob(self, err, blob_hash):
         log.warning("Can't reflect blob %s", str(blob_hash)[:16])
```

Comment:
can you log the err traceback so that we know why this happens? I do worry that maybe we'll continue retrying something ad-infinitum without reason.